modules/logging-loki-storage-aws.adoc

```diff
@@ -25,3 +25,23 @@ $ oc create secret generic logging-loki-aws \
   --from-literal=access_key_secret="<aws_access_key_secret>" \
   --from-literal=region="<aws_region_of_your_bucket>"
 ----
+
+== AWS storage for STS enabled clusters
+
+If your cluster has STS enabled, the Cloud Credential Operator (CCO) supports short-term authentication using AWS tokens.
+
+.Required data
```

Comment:
If we are adding the attributes on CCO generated secret for AWS, I would suggest to add for Azure as well. Azure section seem to be missing this.